woocommerce/woocommerce
# Turn WC_Order::get_tax_location public

This PR turns the `get_tax_location` function public, this is needed so plugins like Avatax can get the tax location to recalculate taxes (for subscriptions for example). An alternative would be to duplicate the function.

### How to test the changes in this Pull Request:

This is a safe backward compatible change
1. Go to an on hold order in WooCommerce admin.
2. Change an item price.
3. Recalculate totals, it should work fine.



### Other information:




### FOR PR REVIEWER ONLY:
